episodes/chap4_manage_via_ui.md

```diff
@@ -135,9 +137,9 @@ You will see a message:
 
 You will need the API keys when you run the nodes to authenticate with the vantage6 server. We will discuss this in [Chapter 6](./chap6_setup_node.md).
 
-1. After creating the collaboration, you can see the details of the collaboration by clicking on the specific collaboration listed in the `Collaborations` tab.
+5. After creating the collaboration, you can see the details of the collaboration by clicking on the specific collaboration listed in the `Collaborations` tab.
    - You will see what organizations are participating in the collaboration.
-   - You will see the nodes of the organizations that are registered in the collaboration. If no nodes are registered, you can register them manually by clicking the `Register missing nodes` button.
+   - You will see the nodes of the organizations that are registered in the collaboration. In case one or more organizations did not register their nodes when creating or updating the collaboration, you can register them manually by clicking the `Register missing nodes` button.
```

Comment:
Maybe note that the button is not visible if no nodes are missing (otherwise participants may wonder where the button is)